src/java.base/share/classes/java/nio/file/attribute/FileTime.java

```diff
@@ -232,36 +232,28 @@ public Instant toInstant() {
             long secs = 0L;
             int nanos = 0;
             switch (unit) {
-                case DAYS:
-                    secs = scale(value, SECONDS_PER_DAY,
-                                 Long.MAX_VALUE/SECONDS_PER_DAY);
-                    break;
-                case HOURS:
-                    secs = scale(value, SECONDS_PER_HOUR,
-                                 Long.MAX_VALUE/SECONDS_PER_HOUR);
-                    break;
-                case MINUTES:
-                    secs = scale(value, SECONDS_PER_MINUTE,
-                                 Long.MAX_VALUE/SECONDS_PER_MINUTE);
-                    break;
-                case SECONDS:
-                    secs = value;
-                    break;
-                case MILLISECONDS:
+                case DAYS    -> secs = scale(value, SECONDS_PER_DAY,
+                        Long.MAX_VALUE / SECONDS_PER_DAY);
+                case HOURS   -> secs = scale(value, SECONDS_PER_HOUR,
+                        Long.MAX_VALUE / SECONDS_PER_HOUR);
+                case MINUTES -> secs = scale(value, SECONDS_PER_MINUTE,
+                        Long.MAX_VALUE / SECONDS_PER_MINUTE);
```

Comment:
It would be nicer to keep the second line aligned with the opening parenthesis, as it was before.

Reply:
Change made in commit 2f179b5, but reverted all changes made to file as per Alan's request. See 23f53c5